### PR TITLE
[Doc] Optimize pulsar-manager documentation

### LINKED ISSUE #13583: [Doc] pulsar-manager installation and deployment documentation optimization sequence
### What issue do you find in Pulsar docs?

Currently, when deployed according to the documentation, the structure is confusing and not easy to install quickly.

### What is your suggestion?

The overall installation and deployment documentation is based on the docker method of deployment, with other installation methods as supplementary instructions. 
The general idea is as follows:

- 1 Easy installation based on docker image
  - 1.1 If database and JWT authentication are not required, install the following.
     Install Pulsar Manager
  - 1.2 If you need to configure database or JWT authentication, add the following steps.
     Configure the database
     Enable JWT
  - 1.3 Set the administrator account and password
  - 1.4 Configure the environment
     Login to the system Visit http://localhost:9527 to login.
- 2 Other Installation
   Bare-metal installation and custom docker image installation
- 3 Configuration instructions

### Do you have any references?

no

### Would you like to fix this issue?

I can

### Note

- [X] I have researched my question.

### PR DESCRIPTION
**Pull request by fanla2021**
Fixed #13583
### Motivation

Currently, when deployed according to the documentation, the structure is confusing and not easy to install quickly.

### Modifications

I have modified the structure of the documentation to follow the Docker deployment style to make it easier for users to understand and deploy quickly.
8e4ce500e98c5a393bfe9cfbe859925
The above picture is the revised outline of the article.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes /**no**)
  - The public API: (yes / **no**)
  - The schema: (yes / **no** / don't know)
  - The default values of configurations: (yes / **no**)
  - The wire protocol: (yes / **no**)
  - The rest endpoints: (yes / **no**)
  - The admin cli options: (yes / **no**)
  - Anything that affects deployment: (yes / no/ **don't know**)

### Documentation

Need to update docs? 

- [ ] `doc-required`   
- [ ] `no-need-doc`   
- [x] `doc`

### CODE REVIEW

**Comment by fanla2021** [comment]
@Anonymitaet
 I have done this part of the revision, please help you review.

**Comment by Anonymitaet** [comment]
Hi @tuteng can you review this from the technical perspective? After that, I'll review it from the technical writing perspective, thanks!

**Comment by fanla2021** [comment]
> can you review this from the technical perspective? After that, I'll review it from the technical writing perspective, thanks!

@tuteng Please help to confirm as soon as possible, thanks!

File: site2/docs/administration-pulsar-manager.md [hunk 1]
```diff
@@ -21,89 +22,42 @@ docker run -it \
...
-./bin/pulsar-manager --redirect.host=http://localhost --redirect.port=9527 insert.stats.interval=600000 --backend.jwt.token=token --jwt.broker.token.mode=PRIVATE --jwt.broker.private.key=file:///path/broker-private.key --jwt.broker.public.key=file:///path/broker-public.key 
+  3. Add a configuration mount and start with a docker image.
+```bash
+docker pull apachepulsar/pulsar-manager:v0.2.0
+docker run -it \
+    -p 9527:9527 -p 7750:7750 \
+    -v /your-path/application.properties:/pulsar-manager/pulsar-
+manager/application.properties
+    -e SPRING_CONFIGURATION_FILE=/pulsar-manager/pulsar-manager/application.properties \
+    apachepulsar/pulsar-manager:v0.2.0
```
Comment (tuteng) [inline site2/docs/administration-pulsar-manager.md:53]:
```
docker run -it \
    -p 9527:9527 -p 7750:7750 \
    -v /your-path/application.properties:/pulsar-manager/pulsar-manager/application.properties \
    -e SPRING_CONFIGURATION_FILE=/pulsar-manager/pulsar-manager/application.properties \
    apachepulsar/pulsar-manager:v0.2.0
```
Should this command look like this?

File: site2/docs/administration-pulsar-manager.md [hunk 2]
```diff
@@ -114,69 +68,116 @@ or
...
-docker run -it -p 9527:9527 -p 7750:7750 -e REDIRECT_HOST=http://localhost -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -v $PWD:/data apachepulsar/pulsar-manager:v0.2.0 /bin/sh
+```bash
+docker pull apachepulsar/pulsar-manager:v0.2.0
+docker run -it \
+    -p 9527:9527 -p 7750:7750 \
+    -v /your-path/application.properties:/pulsar-manager/pulsar-
+manager/application.properties
+    -v /your-path/private.key:/pulsar-manager/private.key
+    -e SPRING_CONFIGURATION_FILE=/pulsar-manager/pulsar-manager/application.properties \
+    apachepulsar/pulsar-manager:v0.2.0
```
Comment (tuteng) [inline site2/docs/administration-pulsar-manager.md:88]:
```
docker run -it \
    -p 9527:9527 -p 7750:7750 \
    -v /your-path/application.properties:/pulsar-manager/pulsar-manager/application.properties \
    -v /your-path/private.key:/pulsar-manager/private.key \
    -e SPRING_CONFIGURATION_FILE=/pulsar-manager/pulsar-manager/application.properties \
    apachepulsar/pulsar-manager:v0.2.0
```

Should this command look like this?

**Comment by Anonymitaet** [comment]
Hi @fanla2021 any thoughts on the suggestions?

File: site2/docs/administration-pulsar-manager.md (hunk 1 above)
Comment (fanla2021) [reply site2/docs/administration-pulsar-manager.md:53]: That's it. I've simplified the Quick Start command, if you need to set environment variables, you can refer to the detailed parameters below.

File: site2/docs/administration-pulsar-manager.md (hunk 2 above)
Comment (fanla2021) [reply site2/docs/administration-pulsar-manager.md:88]: That's it. I've simplified the Quick Start command, if you need to set environment variables, you can refer to the detailed parameters below.

**Comment by fanla2021** [comment]
> any thoughts on the suggestions?

@Anonymitaet @tuteng Can this pr be advanced?

File: site2/docs/administration-pulsar-manager.md [hunk 3]
```diff
@@ -114,69 +68,116 @@ or
...
-```
-export JWT_TOKEN="your-token"
-docker run -it -p 9527:9527 -p 7750:7750 -e REDIRECT_HOST=http://localhost -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -v $PWD:/data apachepulsar/pulsar-manager:v0.2.0 /bin/sh
+```bash
+docker pull apachepulsar/pulsar-manager:v0.2.0
+docker run -it \
+    -p 9527:9527 -p 7750:7750 \
+    -v /your-path/application.properties:/pulsar-manager/pulsar-
+manager/application.properties
+    -v /your-path/private.key:/pulsar-manager/private.key
```
Comment (Anonymitaet) [inline site2/docs/administration-pulsar-manager.md:86]: @fanla2021 seems that you do not incorporate @tuteng's suggestions (lack of \)?

Comment (fanla2021) [reply site2/docs/administration-pulsar-manager.md:86]: @Anonymitaet @tuteng I think the current docker quick start configuration is reasonable, and the complex configuration can be done by referring to the configuration notes below.

**Review by tuteng — APPROVED** [review]

File: site2/docs/administration-pulsar-manager.md [hunk 4]
```diff
@@ -114,69 +68,116 @@ or
...
+  
+  > If you don't want to modify the `application.properties` file, you can add the configuration to the startup parameters via `. /bin/pulsar-manager --backend.jwt.token=token` to add the configuration to the startup parameters. This is a capability of the spring boot framework.
 
-[Set the administrator account and password](#set-administrator-account-and-password).
+- Start Pulsar Manager 
+  ```bash
+  ./bin/pulsar-manager 
+  ```
+
+ ### custom docker image installation
```
Comment (Anonymitaet) [inline site2/docs/administration-pulsar-manager.md:150]:
```suggestion
 ### Custom Docker image installation
```

**Comment by Anonymitaet** [comment]
Hi @fanla2021 can you resolve conflicts?

**Comment by fanla2021** [comment]
> Hi @fanla2021 can you resolve conflicts?

I have finished dealing with the conflict, please confirm

**Pull request merged**